web/themes/custom/votegov/templates/block/block-content--search.html.twig

```diff
@@ -23,12 +28,11 @@
   <input class="usa-input" data-test="search-box" id="search-field-en-small" type="search" name="query"
          autocomplete="off"
          placeholder="{{ content.field_search_placeholder | field_value }} {{ affiliated_search != '1' ? '(in English)' | t }}">
-
-  <button class="usa-button" type="submit">
-    <img
-      src="{{ uswds_img_path }}/usa-icons-bg/search--white.svg"
-      class="usa-search__submit-icon"
-      alt="{{ 'Search Vote.gov' | t }}"
-    />
-  </button>
+  <div class="arrow-icon" role="button" type="submit" tabIndex="0">
```

Comment:
We need the markup to be accessible and submit the form. Let's replace the arrow-icon div with:
`<button class="usa-button usa-button--unstyled usa-button--search" type="submit">`
You can target the `usa-button--search` class for any custom styling.